tempodb/encoding/vparquet3/schema.go

```diff
@@ -252,29 +268,96 @@ type Trace struct {
 
 func attrToParquet(a *v1.KeyValue, p *Attribute, counter droppedAttrCounter) {
 	p.Key = a.Key
-	p.Value = nil
-	p.ValueBool = nil
-	p.ValueDouble = nil
-	p.ValueInt = nil
+	p.ValueType = 0
+	p.Value = p.Value[:0]
```

Comment:
I think the current test had two weak spots:
* The round trip approach can cover some values bleeding through. For example if the value type of an attribute is `attrTypeString` it will ignore any double or int values that are not reset properly
* The simple trace does not have many attributes with a diverse range of types, this makes it less likely to discover values bleeding through on reused attributes

I tried to address both issues and I think the new version of the test works quite well

Reply:
changes look good. thank you!